src/components/AccessKeyForm.js

```diff
@@ -149,7 +149,7 @@ const AccessKeyForm = ({ initialAccountId = null, minimal = false }) => {
           if (
             err &&
             err.hasOwnProperty("message") &&
-            "message" === typeof err.message
+            "string" === typeof err.message
```

Comment:
Problem with this is it makes secret key error look bad, since now it is using the message property of error, not data, which is the human readable one, but was slug-cased in the decryption key error.
![Screenshot 2022-10-10 122027](https://user-images.githubusercontent.com/1994311/194911621-b310e08c-0f7c-4e2a-ac22-b4830ddd9dc5.jpg)

Reply:
Ideally, it'd use `data` first, then message.

```
{
    "code": 406,
    "message": "no_secret_ids",
    "data": "No secret ids found"
}
```

Reply:
@Shelob9 Yes, but beacuse message `data` is null, it will fall back to `message` which solves problem.

Reply:
2f8036446c5cceca059d5a241abe42af0efd56f7da94107ffeb91cfd6a948d8f